lua/orgmode/config/defaults.lua

```diff
@@ -175,6 +175,9 @@ local DefaultConfig = {
     menu = {
       handler = nil,
     },
+    virtual_indent = {
```

Comment:
Why would we need a custom handler for this?

Reply:
For example, if the user wants to make virtual indents with indent lines (something like [`indent-blankline.nvim`](https://github.com/lukas-reineke/indent-blankline.nvim)). Or change the size of the indentation. Or make the indentation colored depending on its size or title. It is quite difficult to cover all the options, so the user can provide his own handler.

Reply:
Let's remove this option for the time being. Once the feature is officially released and there is a proper Neovim release with the support, we can expand on it.